src/components/quickTransfer/quickTransfer.css

```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap');
+
+.transfer-box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  background-color: #f9f9f9;
+  font-family: 'Inter', sans-serif;
```

Comment:
I have added the font-family to index.css , i think it is better to move this to there since it used for most part of the components